app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,15 @@
   </head>
 
   <body>
+    <nav class="navbar navbar-default" role="navigation">
+      <div class="navbar-header">
+        <% if !current_user %>
+          <%= link_to "Log In with Google", google_login_path, class: "navbar-brand" %>
```

Comment:
We figured we can use styling later to have a logo on the left for the home button, and the login/out on the right-hand side.

Reply:
in Ruby, `if !` should be replaced with `unless` as a standard preference.

Reply:
Thanks for the insight, Ian. We just changed the conditional to `unless`.